src/daemonizer/CMakeLists.txt

```diff
@@ -54,6 +54,10 @@ else()
   )
 endif()
 
+#if(CMAKE_BUILD_TYPE STREQUAL "Debug")
+#  add_definitions(-DDEBUG_TMPDIR_LOG=1)
+#endif()
+
```

Comment:
Why this define is commented?

Reply:
I'm using debug build on our servers and I don't want this log even in debug build. Code kept in order to have less conflicts with upcoming merge